README_es.md

```diff
@@ -0,0 +1,333 @@
+    },
+    ...
+  ]
+}
+```
+## 3. Run Scripts
+### 3.1 Run Finetuning
+
+Puedes ejecutar `scripts/run_finetune.sh` para ajustar finamente un modelo base GPT-2.
+```sh
+./scripts/run_finetune.sh
+```
+
+Si deseas proporcionar argumentos para DeepSpeed que reflejen la configuración de tu máquina, puedes pasar los argumentos correspondientes a DeepSpeed al script. Por ejemplo:
+```sh
+./scripts/run_finetune.sh "--num_gpus=8 --master_port 10001"
+```
+
+Para habilitar el ajuste fino de LoRA, puedes consultar:
+```sh
+./scripts/run_finetune_with_lora.sh
+```
+which can be run in similar manner.
+
+Para obtener configuraciones detalladas, uno puede modificar estos scripts directamente. Estos scripts en realidad solo llaman al script de Python examples/finetune.py, que se puede ejecutar de la siguiente manera:
+
+```sh
+deepspeed ${deepspeed_args} \
+  examples/finetune.py \
+    --deepspeed configs/ds_config_zero3.json \
+    --bf16 \
+    --run_name finetune_with_lora \
+    --model_name_or_path facebook/galactica-1.3b \
+    --num_train_epochs 0.01 \
+    --learning_rate 2e-5 \
+    --dataset_path ${dataset_path} \
+    --per_device_train_batch_size 1 \
+    --per_device_eval_batch_size 1 \
+    --validation_split_percentage 0 \
+    --logging_steps 20 \
+    --block_size 512 \
+    --do_train \
+    --output_dir output_models/finetune \
+    --overwrite_output_dir \
+    --ddp_timeout 72000 \
+    --save_steps 5000 \
+    --dataloader_num_workers 1
+```
+
+```python
+python examples/finetune.py -h
+```
+
+### 3.2 Run Evaluation
+
+Uno puede ejecutar la evaluación directamente con un modelo existente de Hugging Face, por ejemplo, para ejecutar GPT2 large, se puede ejecutar:
+```sh
+./scripts/run_evaluation.sh
+```
+
+Para modelos ajustados finamente con LoRA, uno puede consultar:
+```sh
+./scripts/run_evaluation_with_lora.sh
+```
+
+`tests`.
+Esos scripts invocan los ejemplos `examples/*.py` construidos sobre nuestras APIs. Para obtener más ejemplos relacionados con las APIs, uno puede consultar los métodos en el unittest `tests`.
+
+
+## 4. Additional Notes
+### 4.1 LLaMA Checkpoint
+
+1. Primero, necesitas obtener el acceso al modelo LLaMA desde [facebookresearch/llama](https://github.com/facebookresearch/llama). Descarga los checkpoints oficiales y guárdalos en ${llama-path}.
+
+2. Segundo, convierte los checkpoints oficiales `${llama-path}` a checkpoints compatibles con HuggingFace $`${llama-hf-path}` ejecutando:
+
+    `python ./scripts/convert_llama_weights_to_hf.py --input_dir ${llama-path} --model_size 7B --output_dir ${llama-hf-path}/llama-7b-hf`
+
+3. ¡Listo! Ahora puedes establecer la ruta del checkpoint en `${llama-hf-path}/llama-7b-hf`. ¡Disfrútalo!
+
+
+### 4.2 DeepSpeed Config
+
+Puedes configurar DeepSpeed en configs. Los detalles se pueden consultar en [Configuración de DeepSpeed](https://www.deepspeed.ai/docs/config-json/).
+
+
```

Comment:
Add a new Note for "Our support for Mac OS system is not been thoroughly tested yet."
For more information see [this issue](https://github.com/OptimalScale/LMFlow/issues/64#issuecomment-1495511446)